.github/workflows/pythonapp.yml

```diff
@@ -205,16 +205,13 @@ jobs:
     strategy:
       matrix:
         environment:
-          - "PT16+CUDA102"
```

Comment:
Hi @wyli , thanks for the quick fix, may I know the strategy of PT tests? Only maintain the latest 6 environments?

Thanks.

Reply:
the premerge ones test both docker + pip version, currently it's set to run 6 different versions

Reply:
I see.

Thanks.